near/res/testnet_genesis_config.json

```diff
@@ -1,6 +1,7 @@
 {
   "protocol_version": 4,
   "config_version": 1,
+  "genesis_height": 0,
```

Comment:
You shouldn't manually modify config ever. Instead you should use migration script that modifies the config and removes the field at the rollback. @ailisp

Reply:
Same here. Before we figure out the release process it doesn't make sense to have a migration script.

Reply:
It's important to do this before we do the next release. We also have a script already that works and can be used especially for what you're doing.
It just reads the JSON file from the old expected location instead of pulling it from s3. Changing it would be trivial.